oml/functional/metrics.py

```diff
@@ -297,6 +494,38 @@ def _to_tensor(array: Union[np.ndarray, torch.Tensor]) -> torch.Tensor:
         raise TypeError("Wrong type")
 
 
+def _check_if_nonempty_integers_and_positive(seq: Sequence[int], name: str) -> None:
```

Comment:
let's move it to `oml.utils.misc` and remove the underscore from the name
what do you think?

Reply:
I don't know whether this function could be useful in some other places, except functional/metrics.py.

Reply:
At least here: https://github.com/OML-Team/open-metric-learning/blob/b25860c9f6018f85153963f270b3ecdb43ffc08d/oml/metrics/embeddings.py#L250, https://github.com/OML-Team/open-metric-learning/blob/b25860c9f6018f85153963f270b3ecdb43ffc08d/oml/metrics/embeddings.py#L256,

Reply:
Sorry, but I don't see where `_check_if_nonempty_integers_and_positive` could be used in `get_plot_for_queries`. I only see some conditions on storage keys. What variable should contain positive integers only? Or you want to check that `query_ids` is not empty and has some indices?

Reply:
I am talking about `n_queries` and `n_instances`